src/io.jl

```diff
@@ -497,78 +431,55 @@ Currently, grid voltages and line properties are ignored. This function places a
 voltage values.
 """
 function to_pandapower(grid::Grid,)
-    const LOAD_VOLT = 110 # kV # This is a crude approximation
-    const GEN_VOLT = 380 # kV # This is a crude approximation
+    LOAD_VOLT = 110 # kV, this is a crude approximation
+    GEN_VOLT = 380 # kV, this is a crude approximation
     voltage(bus::LoadBus) = LOAD_VOLT
     voltage(bus::GenBus) = GEN_VOLT
+
     # Create empty pandapower network
-    pgrid = pycall(pp[:create_empty_network], PyObject)
+    pgrid = pycall(pp.create_empty_network, PyObject)
+
     # Create buses
     for i in 1:length(buses(grid))
-        pp[:create_bus](
+        pp.create_bus(
             pgrid,
             voltage(buses(grid)[i]),
             index = (i - 1), # Adopting zero indexing because this is for Python.
             geodata = (buses(grid)[i].coords.lat, buses(grid)[i].coords.lon)
         ) # Despite what the documentation says, 'create_bus' requires voltage.
     end
+
     # Create loads and generators
     for i in 1:length(buses(grid))
         if isa(buses(grid)[i], LoadBus)
-            pp[:create_load](
+            pp.create_load(
                 pgrid,
                 bus = (i-1), # Adopting zero indexing because this is for Python.
-                p_kw = 1000*buses(grid)[i].load, # MW to kW
+                p_mw = buses(grid)[i].load, # MW to kW
                 controllable = false # for OPF
             )
         else
         # The type of the generator is being stored in 'name' instead
         # of in 'type' because Julia seems to have problems with a parameter called 'type'.
             for gen in buses(grid)[i].gens
-                pp[:create_gen](
+                pp.create_gen(
                     pgrid,
                     bus = (i-1), # Adopting zero indexing because this is for Python.
-                    p_kw = -1000 * gen.cap, # negative by pandapower convention
+                    p_mw = gen.cap * -1,  # Negative for generation
                     name = gen.tech,
-                    max_p_kw = -1000 * gen.minload, # min inverts with max because negative
-                    min_p_kw = -1000 * gen.cap,
+                    max_p_mw = gen.minload * -1,  # Negative for generation
```

Comment:
it is not just a matter of being negative here. Besides flipping the sign, we are inverting `min` and `max`.